interoperability/upscaling.md

```diff
@@ -0,0 +1,32 @@
+| UPSC-05 | The selected platform should support the provisioning of dedicated compute power and storage service for the upscaling scenario to be executed. For both openEO-based and Application Package-based algorithms, Platform Providers can choose whether to offer this through dedicated capacity or by utilising a large pool of elastic capacity.  | Ensuring necessary resources for larger upscaling tasks is crucial. For Application Package-based upscaling, provisioning can include:                                                                                                                                                                                                                           |
+|         |                                                                                                                                                                                                                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                  |
+|         |                                                                                                                                                                                                                                                                                                                                                   | * Provisioning of dedicated Cloud processing resources (for instance, Kubernetes dedicated PODs) for the hosting of the algorithm(s) selected by the project for the processing of large-scale geographical areas.                                                                                                                                               |
+|         |                                                                                                                                                                                                                                                                                                                                                   | * Provisioning of dedicated HPC resources when available. For such environments, Toil is the recommended workflow engine optimised for distributed computing environments, enabling the execution of CWL workflows across multiple compute nodes.                                                                                                                |
```

Comment:
Doesn't it go quite far to recommend a workflow engine in requirements that are supposed to be about technology agnostic standards?
If we want to give suggestions on how to achieve the requirements, maybe links to the EOEPCA project can help, because they (are supposed to) maintain documentation on best practices.